### PR TITLE
Feat:"added Wishlist Page"

### PR DESCRIPTION
**Pull request by sadathshariff**
Added My Wishlist Page
E-yard MyWishlist

### CODE REVIEW

**Review by SimranjitKaurMaan — APPROVED** [review]
Looks good to merge.

File: Pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>E-Yard | My Wishlist</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.12.1/css/all.css"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../CSS/CommonStyles/style.css" />
+    <link rel="stylesheet" href="../CSS/wishlist.css" />
```
Comment (shuntzhere) [inline Pages/wishlist.html:13]: Is it possible to import wishlist.css to style.css. It looks like your style.css is your main css file.

File: Pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  </head>
+  <body>
+    <header class="header navbar-container">
+      <h1 class="headline-2">
+        <a href="">E-Yard</a>
+      </h1>
```
Comment (shuntzhere) [inline Pages/wishlist.html]: Give this href = '#'

**Review by shuntzhere — COMMENTED** [review]
The html is very neat. The classes are named well too. Just suggested a view things, do check out.

File: Pages/wishlist.html (hunk 2 above)
Comment (sadathshariff) [reply Pages/wishlist.html]: Thanks for pointing it out, will do that

File: Pages/wishlist.html (hunk 1 above)
Comment (sadathshariff) [reply Pages/wishlist.html:13]: I have separated the folder where card and navbar will be in common styles and wishlist will be separate

**Pull request merged**